gradle/scripts/ember.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: "com.moowork.node"
 
 node {
   // Version of node to use.
-  version = '7.8.0'
+  version = '7.10.1'
```

Comment:
Current set of deps defined in package.json doesn't work with v8 yet. Will let Seyi do the update.